core/profiles/standard/standard.install

```diff
@@ -369,7 +369,7 @@ function standard_install() {
       'body' => array(
         LANGUAGE_NONE => array(
           array(
-            'value' => st('This is your first post! You may edit or delete it.'),
+            'value' => '<p>' . st('This is your first post! You may edit or delete it.') . '</p>',
```

Comment:
Because CKEditor 5 requires a wrapping tag, this change prevents the CKEditor 5 "Content will be reformatted" warning on the out of box post and page nodes.